src/vitabel/timeseries.py

```diff
@@ -65,10 +65,22 @@ def _timeseries_list_info(series_list: list[TimeSeriesBase]) -> pd.DataFrame:
                 "Offset": series.offset,
             }
         )
+
+        if hasattr(series, "anchored_channel"):
+            if series.anchored_channel is not None:
+                info_dict[idx]["Attached Channel"] = series.anchored_channel.name #name of the channel
+            else:
+                info_dict[idx]["Attached Channel"] = ""
+        if hasattr(series, "labels"):
+            info_dict[idx]["Attached Labels"] =  ", ".join(
+                [label.name for label in series.labels]
+            )
+
     df = pd.DataFrame(info_dict).transpose()
     df_columns = ['Name', 'Length', 'First Entry', 'Last Entry', 'Offset']
-    metadata_columns = [col for col in df.columns.values if col not in df_columns]
-    return df.reindex(columns=df_columns + metadata_columns)
+    attach_columns = ["Attached Channel", "Attached Labels"]
+    metadata_columns = [col for col in df.columns.values if col not in df_columns+ attach_columns]
+    return df.reindex(columns=df_columns + metadata_columns + [col for col in attach_columns if col in df.columns])
```

Comment:
this is a bit hard to read. as metadata can potentially more complex I think it would make more sense to keep it as the last column; then we could just append them (if necessary) to `df_columns` directly. I'll push a commit with this proposal.